.github/workflows/e2e-full.yml

```
@@ -46,6 +46,7 @@ jobs:
         with:
           using: ${{ matrix.external_net }} ${{ matrix.globalnet }}
           skip: ""  # Override skipping external network tests
+          plugin: "scripts/e2e/external/hook"
```

```suggestion
          plugin: scripts/e2e/external/hook
```

No need to quote this